src/main/archetype/ui.content/src/main/content/jcr_root/conf/__confFolderName__/settings/wcm/policies/.content.xml

```diff
@@ -12,7 +12,7 @@
                         jcr:primaryType="nt:unstructured"
                         jcr:title="${siteName} Default"
                         sling:resourceType="wcm/core/components/policy/policy"
-                        components="[group:${componentGroupName},/apps/${appsFolderName}/components/form/container]"
+                        components="[group:${componentGroupName},/apps/${appsFolderName}/components/form/container,/libs/dam/cfm/components/contentfragment,/libs/cq/experience-fragments/editor/components/experiencefragment,/libs/wcm/foundation/components/responsivegrid]"
```

Comment:
Instead of OOTB CF, should we use CF from Core component?
Might need to bump up the core component version.

Reply:
@gabrielwalt Please confirm

Reply:
@bpauli Please confirm query related to Content Fragment component.